src/server.js

```diff
@@ -0,0 +1,36 @@
+// set up express
+const express = require("express");
+const fs = require("fs");
+const path = require("path");
+// connect my routes to this file
+
+const apiRouter = require("../routes/apiRoutes");
```

Comment:
again, really neat imports :-)

I like that you have separated importing external libraries from the code (ie the router) that you have written yourself - great stuff